kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -736,6 +740,40 @@ class KRATOS_API(KRATOS_CORE) SimpleMortarMapperProcess
                             #pragma omp atomic
                             r_nodal_area += rThisMortarOperators.DOperator(i_node, i_node);
                         }
+                        // In case of discontinous interface we add contribution to near nodes
+                        if (mDiscontinuousInterface) {
+                            const double element_length = r_slave_geometry.Length();
+
+                            // Iterating over nodes
+                            for (IndexType i_node = 0; i_node < TNumNodes; ++i_node) {
+                                const double nodal_area_contribution = rThisMortarOperators.DOperator(i_node, i_node);
+
+                                // The original node coordinates
+                                const array_1d<double, 3>& r_slave_node_coordinates = r_slave_geometry[i_node].Coordinates();
+
+                                // Iterating over other paired conditions
+                                const auto& index_masp_master = p_cond_master->GetValue(INDEX_SET);
```

Comment:
```suggestion
                                const auto &r_index_masp_master = p_cond_master->GetValue(INDEX_SET);
```

Reply:
I am on the type& var side, not the type &var side